types/v56/vm_types.go

```diff
@@ -161,3 +160,46 @@ type SourcedVmTemplateParams struct {
 	VmTemplateInstantiationParams *InstantiationParams `xml:"VmTemplateInstantiationParams,omitempty"` // Same as InstantiationParams used for VMs within a vApp
 	StorageProfile                *Reference           `xml:"StorageProfile,omitempty"`                // A reference to a storage profile to be used for the VM. The specified storage profile must exist in the organization vDC that contains the composed vApp. If not specified, the default storage profile for the vDC is used.
 }
+
+type OVF_Environment struct {
```

Comment:
I would like to see descriptions of what the fields mean.
Where did you get the data structure? Could you share a link to the API element you have used?